### ISSUE TITLE
Docker module choose random port if we didn't pass a variable

### ISSUE THREAD

**Issue by vankhoa011** [ansible/ansible-modules-core#1185]
In 1.8.0 version, If I didn't pass "ports" variable, Docker module will not bind any port to the Docker Host. It worked correctly.
However, In 1.9.0.1, Docker module choose a random ports, and bind to Docker Host. I think there is something wrong.
If I remove this line ( ports: "{{ docker_ports|default(omit) }}" ) , the docker module works correctly.

```
  - name: Run docker container
     docker:
       image: "{{ docker_image }}"
       name: "{{ docker_name }}"
       command: "{{ docker_command|default(omit) }}"
       ports: "{{ docker_ports|default(omit) }}"
       env: "{{ docker_env|default(omit) }}"
       volumes: "{{ docker_volumes|default(omit) }}"
```

Move issue from ansible to ansible modules core , follow this link.
https://github.com/ansible/ansible/issues/10592

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by chouseknecht** [comment]
wontfix

Docker modules have been refactored. If this is still an issue after testing with the Ansible 2.1.1.0, please open a new issue.
